src/openrct2/Context.cpp

```diff
@@ -164,7 +171,7 @@ namespace OpenRCT2
             //       If objects use GetContext() in their destructor things won't go well.
 
             GameActions::ClearQueue();
-            network_close();
+            _network.Close();
```

Comment:
I don't understand how this change is  tied to `https://github.com/OpenRCT2/OpenRCT2/pull/15232/commits/40313b1ea3182adbb4586e9c8ca13032a19a0350`